src/OpenLoco/Windows/ToolbarTop.cpp

```diff
@@ -329,27 +331,28 @@ namespace OpenLoco::Ui::Windows::ToolbarTop::Game
     static void railroadMenuMouseDown(Window* window, WidgetIndex_t widgetIndex)
     {
         // Load dropdown objects removing any that are not unlocked.
-        registers regs;
-        regs.edi = X86Pointer(&available_objects[0]);
-        call(0x004A6841, regs);
-
-        // Sanity check: any objects available?
-        uint32_t i = 0;
-        while (available_objects[i] != 0xFF && i < std::size(available_objects))
-            i++;
-        if (i == 0)
+        // Note: This is not using player company id! This looks odd.
```

Comment:
that does sound odd, does it feel like an original bug/typo? does the game crash if you use the player company id instead here?

Reply:
This is within ui code so the ai would never call this function. I can't actually think why it could cause an issue just that its a bit odd.

Reply:
yeah that's a strange one then, ah well, I guess we leave it for now